setup.py

```diff
@@ -69,6 +69,7 @@
               'mongodb-to-snowflake=pipelinewise.fastsync.mongodb_to_snowflake:main',
               'mongodb-to-postgres=pipelinewise.fastsync.mongodb_to_postgres:main',
               'mongodb-to-bigquery=pipelinewise.fastsync.mongodb_to_bigquery:main',
+              'partial-mysql-to-snowflake=pipelinewise.fastsync.partialsync.mysql_to_snowflake:main'
```

Comment:
`pipelinewise.fastsync.partialsync.mysql_to_snowflake` module doesn't exist yet, I suggest removing it from this PR.

Reply:
- [ ] Remove it!